packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -1079,16 +1172,97 @@ class ScriptsView implements CoreElementView {
     _items.setRenderer((ScriptRef scriptRef) {
       final String uri = scriptRef.uri;
       final String name = uriDescriber(uri);
-      final CoreElement element = li(text: name, c: 'list-item');
+
+      CoreElement element;
+      if (_matcherRendering != null && _matcherRendering.active) {
+        // InputElement's need to fetch the value not text/textContent property.
+        // The value and text are different, all nodes have a text. It the text
+        // content of the node itself along with its descendants. However, input
+        // elements have a value property - its the input data of the input
+        // element. Input elements may have a text/textContent but it is alway
+        // empty because they are void elements.
+        final html.InputElement inputElement =
+            (_matcherRendering._textfield.element as html.InputElement);
+        final String matchPart = inputElement.value;
+
+        // Bold the characters that match.
+        final int startIndex = name.indexOf(matchPart, 0);
+        final String firstPart = name.substring(0, startIndex);
+        final int endBoldIndex = startIndex + matchPart.length;
+        final String boldPart = name.substring(startIndex, endBoldIndex);
+        final String endPart = name.substring(endBoldIndex);
+        final String autoCompletePart = '$firstPart<b>$boldPart</b>$endPart';
+        element = li(html: autoCompletePart, c: 'list-item');
```

Comment:
nit: use <strong> instead of <b>
Also define the HTML programmatically to avoid XSS or strange rendering when the path looks a bit like html.

Reply:
Using the TreeSanitizer to ensure the HTML is safe.
Also, where did you want 'instead of' to be used?

Reply:
Ironically I got hit by HTML escaping in my comment as well.
```<strong>foo</strong>``` was the tag I'd recommend.

Reply:
done.